.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
```

Comment:
Please don't commit these files. Use git status command, it shows which files you're adding to the commit. Or you can add these to the .gitignore